packages/flutter_tools/lib/src/android/adb.dart

```diff
@@ -204,7 +204,7 @@ class AdbDevice {
   /// Device model; can be null. `XT1045`, `Nexus_7`
   String get modelID => _info['model'];
 
-  set modelID(String value) {
+  void set modelID(String value) {
```

Comment:
I don't mind if we require that it be there and set to void, or require that we omit it.
If you think we should have a lint that requires that we omit it (and have this lint ignore lack of type on setters), then please file a bug here: https://github.com/dart-lang/linter/issues/new
I'm happy to adjust the code the other way once a new lint comes. :-)